README.Rmd

```diff
@@ -12,21 +12,179 @@ knitr::opts_chunk$set(
 [![codecov](https://codecov.io/github/epiverse-trace/bpmodels/branch/main/graphs/badge.svg)](https://codecov.io/github/epiverse-trace/bpmodels) 
+![GitHub contributors](https://img.shields.io/github/contributors/epiverse-trace/bpmodels)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/epiverse-trace/bpmodels)
+![GitHub](https://img.shields.io/github/license/epiverse-trace/bpmodels)
 <!-- badges: end -->
+
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-`bpmodels` is an R package to simulate and analyse the size and length of branching processes with a given offspring distribution.
+`bpmodels` is an R package to simulate and analyse the size and length of 
+branching processes with a given offspring distribution.
 
 # Installation
+
 The latest development version of the `bpmodels` package can be installed via
 
+```{r include=TRUE,eval=FALSE}
+devtools::install_github('epiverse-trace/bpmodels')
+```
+
+To load the package, use
+
+```{r eval=TRUE}
+library('bpmodels')
+```
+
+# Quick start
+
+At the heart of the package are the `chain_ll()` and `chain_sim()` functions. 
+
+## Calculating log-likelihoods
+
+The `chain_ll()` function calculates the log-likelihood of a distribution of 
+chain sizes or lengths given an offspring distribution and its associated 
+parameters. 
+
+If we have observed a distribution of chains of sizes $1, 1, 4, 7$, we can 
+calculate the log-likelihood of this observed chain by assuming the offspring 
+per generation is Poisson distributed with a mean number of $0.5$. 
+
+To do this, we run 
+
+```{r}
+set.seed(13)
+chain_sizes <- c(1, 1, 4, 7) # example of observed chain sizes
+chain_ll(x = chain_sizes, offspring = "pois", stat = "size", lambda = 0.5)
+```
+
+The first argument of `chain_ll()` is the size (or length) distribution to 
+analyse. The second argument, `offspring`, specifies the offspring 
+distribution. This is given as a function used to generate random offspring. 
+It can be any probability distribution implemented in `R`, that is, one that 
+has a corresponding function for generating random numbers beginning with the 
+letter `r`. In the case of the example above, since random Poisson numbers are 
+generated in `R` using a function called `rpois()`, the string to pass to the 
+`offspring` argument is `"pois"`.
+
+The third argument, `stat`, determines whether to analyse chain sizes 
+(`"size"`, the default if this argument is not specified) or lengths 
+(`"length"`). Lastly, any named arguments not recognised by `chain_ll()` 
+are interpreted as parameters of the corresponding probability distribution, 
+here `lambda = 0.5` as the mean of the Poisson distribution (see the `R` help 
+page for the [Poisson distribution](https://stat.ethz.ch/R-manual/R-devel/library/stats/html/Poisson.html) for more information). 
+
+# Imperfect observations
+
+By default, `chain_ll` assumes perfect observation, where `obs_prob = 1` 
```

Comment:
after "assumes perfect observation", you could give a brief explanation, for example: perfect observation (i.e. all transmission events are recorded in the data)

Reply:
Notes. Thanks.